src/components/Table/Table.tsx

```diff
@@ -0,0 +1,107 @@
+import { Spacer } from '@components/Spacer'
+import { Row } from 'simple-flexbox'
+
+import {
+  Body,
+  EmptyMessage,
+  Header,
+  Loader,
+  PageLimit,
+  Pagination
+} from './components'
+import styles from './Table.module.scss'
+import type { ItemBase, TableProps } from './types'
+
+export const Table = <Item extends ItemBase>({
+  name,
+  data,
+  columns,
+  rowActions,
+  options,
```

Comment:
hmm, tak na pierwsze spojrzenie kompletnie nie wiem co to "options" ;p

Reply:
Tak zakładałem, po prostu myślę, że to dobry moment na pomyślenie nad lepszymi nazwami, skoro i tak komponent jeszcze nie jest nigdzie uzywany ; D